src/i18n/de/README.md

```diff
@@ -0,0 +1,89 @@
+# Anleitung für deutsche Übersetzungen
+
+Hallo und herzlich willkommen! Wir freuen uns sehr, dass du dich dafür interessierst, bei der deutschen Übersetzung der Astro-Dokumentation mitzuwirken. 😊🚀
+
+
+## Zielsetzung dieser Anleitung
+
+Wie bei Open Source-Projekten üblich wird unsere Dokumentation von vielen fleißigen Personen auf freiwilliger Basis übersetzt. Die aktuellen Übersetzungen stammen daher aus vielen unterschiedlichen Federn. Zudem ändert sich auch die Besetzung unseres Teams im Laufe der Zeit.
+
+Diese Anleitung soll dazu beitragen, dass sich das Ergebnis beim Lesen trotz aller verschiedenen Einflüsse wie ein Gesamtwerk mit gemeinsamem Schreibstil anfühlt.
+
+&nbsp;
+
+
+## Übersetzungs-Glossar
+
+| Originalbegriff             | Übersetzung                          | Anmerkungen
+|:----------------------------|:-------------------------------------|:------------
+| branch                      | der Branch                           | Wird nicht übersetzt (gängiger Domänenbegriff).
+| build                       | Erzeugung / Erstellung               | Wird aus Konsistenzgründen zu bestehenden Übersetzungen vorerst so übersetzt. Eventuell werden wir "Build" aber zukünftig als gängigen Domänenbegriff einstufen und nicht übersetzen.
+| build process               | Erzeugungs- / Erstellungsvorgang     | s.o.
+| build time                  | Erzeugungs- / Erstellungszeitpunkt   | s.o.
+| to build                    | erzeugen                             | Da die Alternativen "bauen" oder gar "builden" merkwürdig klingen, wird diese Übersetzung vermutlich so bleiben.
+| command line                | die Kommandozeile                    |
```

Comment:
Commit = Commit: Semantisch gibt es keinen deutschen treffenden Begriff, somit würde ich ebenfalls Commit beibehalten. Weiter könnte man generell Begrifflichkeiten aus dem Git-Ökosystem als etablierte Neuologismen (etablierter Neologismus, Kotradiktion in sich selbst lol) etablieren.

Reply:
Good idea! Feel free to make a new PR adding this as this PR has already been merged. :)